tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -27,13 +31,19 @@
 def manual_integration_dir() -> Any:
     """Creates a temporary directory and then deletes it when the calling function is done."""
     if MANUAL_INTEGRATION_TEST:
-        os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = 'path/to/gooogle_api_credential.json'
+        os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = os.path.join(
+            os.environ['HOME'], '.mosaic/mosaicml-research-gcs.json')
```

Comment:
Why did this path change? CI won't have access to this path.

Reply:
this is for local manual testing only.